zombienet/README.md

```diff
@@ -0,0 +1,23 @@
+**Zombienet for macos**
+
+Download from [zombienet](https://github.com/paritytech/zombienet/releases)
+
+or
+
+`wget https://github.com/paritytech/zombienet/releases/download/v1.3.30/zombienet-macos `
+
+`chmod +x zombienet-macos`
+
+`./zombienet-macos --help`
+
+Build Polkadot
+
+`cargo build --release`
+
+Build Foucoco
+
+`cargo build --release`
+
+Run zombienet
```

Comment:
This should have some text that points out that the `zombienet/config.toml` by default looks for the `pendulum-node` binary. If another network is to be used the config needs to be changed accordingly.